packages/stores/src/account/base.ts

```diff
@@ -713,8 +713,8 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
     }
 
     return {
-      accountNumber: Number(account.account_number),
-      sequence: Number(account.sequence),
+      accountNumber: Number(account.accountNumber.toString()),
```

Comment:
What's going on here? Why use `toString()` only to convert to a `Number`?

Reply:
For signing we need to convert these values into numbers. Since The `account` values are both `Int` objects and it lacks a `toNumber()` method, we first need to convert them to strings.

Reply:
Do you mean that `accountNumber` and `sequence` are integers? If so, integer is a subtype of `Number` is it not? And I think JS has no specific `Int` type?

Reply:
We use `Dec` and `Int` objects for handling amounts and mathematical operations on those amounts. They use bignumber (which uses BigInt) under the hood, since raw JS number types have precision issues especially when doing math.